chroma-agent/chroma_agent/device_plugins/linux_components/zfs.py

```diff
@@ -27,13 +27,7 @@
 
 
 def write_to_store(key, value, filename=STORE_FILENAME):
-    data = {}
-
-    try:
-        with open(filename, 'r') as f:
-            data = json.loads(f.read())
```

Comment:
I'm wondering why we are duplicating the store location both within IML and iml-common. Ideally in modules you are encapsulating logic, if you don't you are actually making it harder to use a modular architecture than easier.

Reply:
yes , this needs to be addressed, and will be in #290 + corresponding iml-common patch.